### ISSUE TITLE
validation error on root dataverse, Internal Server Error after logging in when deploying to Payara 5

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#5907]
We're getting a strange error when browsing to the root dataverse when Dataverse is deployed to Payara 5 rather than Glassfish 4.1:

![58833050-3c47f380-861e-11e9-8230-cd1f1c13f3a3](https://user-images.githubusercontent.com/21006/58837007-51c21b00-8628-11e9-9040-c25cb33dc276.png)

The theory by the non-JSF experts (myself included) over at https://github.com/IQSS/dataverse/issues/4172#issuecomment-498426325 is that it has something to do with this:
         
    javax.faces.INTERPRET_EMPTY_STRING_SUBMITTED_VALUES_AS_NULL

Should we try changing it to "false" like this?

```
[pdurbin@tabby dataverse]$ git diff
diff --git a/src/main/webapp/WEB-INF/web.xml b/src/main/webapp/WEB-INF/web.xml
index ff617a9fa..fc9c32ae9 100644
--- a/src/main/webapp/WEB-INF/web.xml
+++ b/src/main/webapp/WEB-INF/web.xml
@@ -42,7 +42,7 @@
         <param-name>
             javax.faces.INTERPRET_EMPTY_STRING_SUBMITTED_VALUES_AS_NULL
         </param-name>
-        <param-value>true</param-value>
+        <param-value>false</param-value>
     </context-param>
     <context-param>
         <param-name>javax.faces.FACELETS_SKIP_COMMENTS</param-name>
[pdurbin@tabby dataverse]$ 
```

**Comment by pdurbin** [comment]
<img width="1263" alt="Screen Shot 2019-06-04 at 12 40 22 PM" src="https://user-images.githubusercontent.com/21006/58897303-ffcfd280-86c5-11e9-8e6e-4c5cd174ccb6.png">

**Comment by scolapasta** [comment]
We definitely  INTERPRET_EMPTY_STRING_SUBMITTED_VALUES_AS_NULL to true purposefully, though I don't recall the exact thing that needed it.(possibly the way we handle "required" validation?)

Regardless a different solution was found, so am closing the PR that changes this.

**Comment by donsizemore** [comment]
Just tried again at @pdurbin's request, deployed Dataverse 4.17 over Payara 5.193.1. The homepage still shows the validation error displayed above. In my case, even though I specify the FAKE DOI provider the first warning in server.log is a login failure:

```
[#|2019-10-28T15:06:39.653+0000|WARNING|Payara 5.193|edu.harvard.iq.dvn.core.index.DOIEZIdServiceBean|_ThreadID=39;_ThreadName=admin-thread-pool::admin-listener(1);_TimeMillis=1572275199653;_LevelValue=900;|
  login failed |#]
[#|2019-10-28T15:06:39.653+0000|WARNING|Payara 5.193|edu.harvard.iq.dvn.core.index.DOIEZIdServiceBean|_ThreadID=39;_ThreadName=admin-thread-pool::admin-listener(1);_TimeMillis=1572275199653;_LevelValue=900;|
  Exception String: edu.ucsb.nceas.ezid.EZIDException: http://goo.gl/EOpFl -->
```

Also at @pdurbin's behest I tried #6220 after I merged it with develop - same behavior as above.

**Comment by poikilotherm** [comment]
Beware... My patch in #6220 does only fix the login issue for the Dataverse ID validation, not for a DOI provider. I did not came across this problem so far. I dunno if I should take a look at this. Depends on @scolapasta... @pdurbin should this be a new issue?

**Comment by scolapasta** [comment]
I'm going to close this, as it is solved in the current version of #6230.
